blueoil/networks/classification/quantize_example.py

```diff
@@ -0,0 +1,147 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.blocks import darknet
+from blueoil.networks.classification.base import Base
+from blueoil.layers import conv2d
+
+
+class FooNetwork(Base):
+    """Example model with simple layer"""
+
+    def __init__(
+            self,
+            *args,
+            **kwargs
+    ):
+        super().__init__(
+            *args,
+            **kwargs,
+        )
+
+        self.activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
+        self.before_last_activation = self.activation
+
+    def base(self, images, is_training):
+        if self.data_format == "NCHW":
+            channel_data_format = "channels_first"
+        elif self.data_format == "NHWC":
+            channel_data_format = "channels_last"
+        else:
+            raise RuntimeError("data format {} should be in ['NCHW', 'NHWC]'.".format(self.data_format))
+
+        self.inputs = self.images = images
+
+        darknet_block = partial(darknet, is_training=is_training,
+                                activation=self.activation, data_format=self.data_format)
+
+        x = darknet_block("block_1", self.inputs, filters=32, kernel_size=1)
```

Comment:
Is it possible not use `darknet_block` ?
Document reader dose not know about `darknet_block`.

Reply:
Now I figured out how to use the ops.

I think I will use blueoil layer(blueoil.layers.conv2d), thus the document reader will follow and use the blueoil layer. What do you think?

Reply:
OK, I see.